From e4c6d162627be8bc30548f9b59f719d4f645c057 Mon Sep 17 00:00:00 2001
From: Richard Hulme <peper03@mythtv.org>
Date: Mon, 13 Apr 2015 20:14:06 +0200
Subject: [PATCH] Don't show the miniplayer in MythMusic when starting the next
 track if we're not playing any more.

This makes playback stop correctly when 'Repeat mode' is off and we've reached the end of the playlist.  Otherwise showing the miniplayer causes playback to start again.
(cherry picked from commit bc1ffba7e48db13db589082968422d02ba0c1fda)
---
 mythplugins/mythmusic/mythmusic/musicplayer.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mythplugins/mythmusic/mythmusic/musicplayer.cpp b/mythplugins/mythmusic/mythmusic/musicplayer.cpp
index 867c765..713408d 100644
--- a/mythplugins/mythmusic/mythmusic/musicplayer.cpp
+++ b/mythplugins/mythmusic/mythmusic/musicplayer.cpp
@@ -504,7 +504,7 @@ void MusicPlayer::nextAuto(void)
     }
 
     // if we don't already have a gui attached show the miniplayer if configured to do so
-    if (m_isAutoplay && m_canShowPlayer && m_autoShowPlayer)
+    if (m_isAutoplay && m_canShowPlayer && m_autoShowPlayer && m_isPlaying)
     {
         MythScreenStack *popupStack =
                             GetMythMainWindow()->GetStack("popup stack");
